R/utils-get_code_dependency.R

```diff
@@ -124,13 +134,36 @@ fix_comments <- function(calls) {
   # If the first token is a COMMENT, then it belongs to the previous call.
   if (length(calls) >= 2) {
     for (i in 2:length(calls)) {
-      if (grepl("@linksto", calls[[i]][1, "text"])) {
+      comments_first <- grepl("@linksto", calls[[i]][1, "text"])
+      comments_second <- grepl("@linksto", calls[[i]][2, "text"]) && calls[[i]][1, "text"] == ""
+      if (comments_first) {
```

Comment:
```suggestion
      comment_idx <- grep("@linksto", calls[[i]][, "text"])
      if (comments_idx) {
```

Reply:
`comments_idx` can be a vector of length > 1, but I see the idea of simplification in here!